blocks/email-settings/edit.tsx

```diff
@@ -92,6 +92,7 @@ export default function Edit() {
       newBlocks = [...parsedContentFromTemplate];
     }
 
+    // @ts-ignore
```

Comment:
comment applies throughout: it's better to use `@ts-expect-error`, so if somehow this line ever stops throwing an error, we'll know about it and can remove the directive